Client/Ecosia/UI/NTP/Cells/NTPBookmarkNudgeCell.swift

```diff
@@ -0,0 +1,182 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Core
+
+final class NTPBookmarkNudgeCell: UICollectionViewCell, NotificationThemeable, ReusableCell {
+    
+    private enum UX {
```

Comment:
Here the approach of `UX` has been followed for some of the elements👌 - for consistency, I'd make some effort to do the same perhaps 🙏